### DISCUSSION TITLE
POST with hx-vals in the body

### DISCUSSION THREAD

**Discussion by florian-sun** [bigskysoftware/htmx#3061] [Q&A]
It seems that the only way to send POST requests with hx-vals as JSON in the body is by using json-enc extension.
Is this correct or do I miss something here?

**Comment by Telroshan** [comment]
Hello, yes, if you want to send values as JSON, you'll need the `json-enc` extension, as htmx will only send form data or URL encoded values by default

**Reply by alirezaalavi87** [reply]
I think this is worth mentioning in the `hx-vals` doc. It's a tricky point to be honest.
